### DISCUSSION TITLE
Can this model be evaluated using ROC curves, calibration curves and DCA curves?

### DISCUSSION THREAD

**Discussion by weiliwei666** [julianspaeth/random-survival-forest#14] [Q&A]
Can random survival forest be evaluated using ROC curves, calibration curves and DCA curves?

or just C-index and brier score?

**Comment by julianspaeth** [comment]
Hi, the package itself only supports the c-index. However, I think using the prediction scores from the `predict()` method on your test data and the corresponding true survival times, you could probably calculate some time-dependent AUC by yourself. If you implement something like that, I would appreciate a pull request such that we can integrate it into the package
